venv/bin/activate

```diff
@@ -0,0 +1,76 @@
+# This file must be used with "source bin/activate" *from bash*
```

Comment:
is this venv for virtual environments to manage package dependencies independent of the os?

Reply:
It depends on operating system. For Mac OS, `venv/bin/activate` or `source venv/bin/activate`. For Windows, `venv\Scripts\activate`.